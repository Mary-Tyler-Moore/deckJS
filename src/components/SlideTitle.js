import React from 'react'
import { Title, Subtitle, Container } from 'bloomer'

const SlideTitle = ({ title, subtitle }) => {
  return (
    <Container className="text">
      <Title className="has-text-weight-light is-size-1 is-size-3-mobile">
        {title}
      </Title>
      <Subtitle className="is-uppercase has-text-weight-light is-size-5 is-size-6-mobile" style={{padding: '.5rem 0 0 .3rem'}}>
        {subtitle}
      </Subtitle>
    </Container>
  )
}

export default SlideTitle
