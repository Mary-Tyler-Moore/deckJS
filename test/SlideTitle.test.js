import React from 'react'
import SlideTitle from '../src/components/SlideTitle'
import { shallow, mount } from 'enzyme'

describe('SlideTitle', () => {
  const mock = {
    title: 'title',
    subtitle: 'subtitle'
  }

  it('should render', () => {
    const wrapper = shallow(<SlideTitle {...mock} />)
    expect(wrapper).toMatchSnapshot()
  })
})