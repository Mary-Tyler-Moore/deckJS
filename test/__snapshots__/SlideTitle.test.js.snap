// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SlideTitle should render 1`] = `
ShallowWrapper {
  "length": 1,
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <SlideTitle
    subtitle="subtitle"
    title="title"
  />,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": undefined,
    "nodeType": "function",
    "props": Object {
      "children": Array [
        <withHelpersModifiers(Title)
          className="has-text-weight-light is-size-1 is-size-3-mobile"
        >
          title
        </withHelpersModifiers(Title)>,
        <withHelpersModifiers(Subtitle)
          className="is-uppercase has-text-weight-light is-size-5 is-size-6-mobile"
          style={
            Object {
              "padding": ".5rem 0 0 .3rem",
            }
          }
        >
          subtitle
        </withHelpersModifiers(Subtitle)>,
      ],
      "className": "text",
    },
    "ref": null,
    "rendered": Array [
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "function",
        "props": Object {
          "children": "title",
          "className": "has-text-weight-light is-size-1 is-size-3-mobile",
        },
        "ref": null,
        "rendered": "title",
        "type": [Function],
      },
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "function",
        "props": Object {
          "children": "subtitle",
          "className": "is-uppercase has-text-weight-light is-size-5 is-size-6-mobile",
          "style": Object {
            "padding": ".5rem 0 0 .3rem",
          },
        },
        "ref": null,
        "rendered": "subtitle",
        "type": [Function],
      },
    ],
    "type": [Function],
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": undefined,
      "nodeType": "function",
      "props": Object {
        "children": Array [
          <withHelpersModifiers(Title)
            className="has-text-weight-light is-size-1 is-size-3-mobile"
          >
            title
          </withHelpersModifiers(Title)>,
          <withHelpersModifiers(Subtitle)
            className="is-uppercase has-text-weight-light is-size-5 is-size-6-mobile"
            style={
              Object {
                "padding": ".5rem 0 0 .3rem",
              }
            }
          >
            subtitle
          </withHelpersModifiers(Subtitle)>,
        ],
        "className": "text",
      },
      "ref": null,
      "rendered": Array [
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "function",
          "props": Object {
            "children": "title",
            "className": "has-text-weight-light is-size-1 is-size-3-mobile",
          },
          "ref": null,
          "rendered": "title",
          "type": [Function],
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "function",
          "props": Object {
            "children": "subtitle",
            "className": "is-uppercase has-text-weight-light is-size-5 is-size-6-mobile",
            "style": Object {
              "padding": ".5rem 0 0 .3rem",
            },
          },
          "ref": null,
          "rendered": "subtitle",
          "type": [Function],
        },
      ],
      "type": [Function],
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactSixteenAdapter {
      "options": Object {
        "enableComponentDidUpdateOnSetState": true,
      },
    },
  },
}
`;
